Implemented CLI option for executing private supermarket profiles (#5749)
chef/inspec: docs-chef-io/content/inspec/cli.md

``` ruby
inspec exec supermarket://username/linux-baseline

> inspec exec supermarket://username/linux-baseline --supermarket_url="https://myprivatesupermarket.mydomain.com"

Reviewer: ```suggestion
inspec exec supermarket://username/linux-baseline --supermarket_url="https://privatesupermarket.example.com"
```

inspec supermarket SUBCOMMAND ...
```

### Options

This subcommand has additional options:

* ``--supermarket_url``

>     Specify private supermarket url to run supermarket commands on private supermarket.

Reviewer: ```suggestion
    Specify the URL of a private Chef Supermarket.
```
